packages/modules/src/modules/banners/designableBanner/components/DesignableBannerArticleCount.tsx

```diff
@@ -33,13 +33,12 @@ export function DesignableBannerArticleCount({
 
 const styles = {
     container: (textColor: string = 'inherit') => css`
-        ${headline.xxxsmall({ fontWeight: 'bold' })}
-        font-size: 15px;
-        color: ${textColor};
         margin: 0;
-
+        color: ${textColor};
+        ${headline.xxxsmall({ fontWeight: 'bold' })};
+        font-size: 94%;
         ${from.desktop} {
-            font-size: 17px;
+            font-size: ${(100 / 94) * 100}%;
```

Comment:
Could you give a little context as to what we're doing here?

It feels like we're saying
- Until desktop, use 94% of the parent
- when desktop, use ~106% of the parent

If that's intended, maybe we could leave a note in the code explaining why as it feels like an edge-case we're creating?

Reply:
Design would like the headline less than desktop here to be 15px. Headline does not provide this size (xxxsmall = 17px). Therefore we either force the font size to 15px (non-accesible, breaking the ability to for the browser to increase font size) or scale it down to 15px (94%). Once we resize above desktop it needs to be reversed, this is what the calculation does.

Reply:
Let me put a comment in the code to this effect,

Reply:
Comment added now.